### PR TITLE
fix: update graphql schema to match docs

### PR DESCRIPTION
**Pull request by jhockett**
Update the one-to-many schema to reflect the newer @connection directive style

#3513 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+
+type Comment @model @key(name: "byPost", fields: ["postID", "content"]) {
   id: ID!
-  content: String
-  post: Post @connection(name: "PostComments")
-}
\ No newline at end of file
+  postID: ID!
+  content: String!
+}
```
Comment (UnleashedMind) [inline packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql]: We should also add the "belongs to" connection into the Comment model definition just to have the same behavior as the previous setup.
`post: Post @connection(fields: ["postID"])`

File: packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 }
-type Post @model {
+
+type Post @model @key(name: "byBlog", fields: ["blogID"]) {
   id: ID!
   title: String!
-  blog: Blog @connection(name: "BlogPosts")
-  comments: [Comment] @connection(name: "PostComments")
+  blogID: String!
+  comments: [Comment] @connection(keyName: "byPost", fields: ["id"])
```
Comment (UnleashedMind) [inline packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql]: We should also add the "belongs to" connection into the Post model definition just to have the same behavior as the previous setup.
blog: Blog @connection(fields: ["blogID"])

Comment (jhockett) [reply packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql]: Changed

File: packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql (hunk 1 above)
Comment (jhockett) [reply packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql]: Changed

**Comment by lawmicha** [comment]
has this been tested with `amplify model codegen`?

**Pull request merged**